### PR TITLE
Add windows Anaconda install to readme.md

### PR DESCRIPTION
**Pull request by Ludee**

### CODE REVIEW

File: snakemake/README.md [hunk 1]
```diff
@@ -10,6 +10,24 @@ Let's cook some Spätzle!
     source /your/path/to/venv/bin/activate
     pip install -r requirements.txt
 
+### Windows
+
+* git bash
+```git 
+cd D:/git/github/rl_institut/
+git clone git@github.com:rl-institut/workshop.git
```
Comment (Bachibouzouk) [inline snakemake/README.md]: this won't work if the user did not set ssh locally, cloning using http is less error prone for new users
```suggestion
git clone https://github.com/rl-institut/workshop.git
```

Comment (nesnoj) [reply snakemake/README.md]: That's right, we may should change this in the linux part too

File: snakemake/README.md [hunk 2]
```diff
@@ -10,6 +10,24 @@ Let's cook some Spätzle!
     source /your/path/to/venv/bin/activate
     pip install -r requirements.txt
 
+### Windows
+
+* git bash
+```git 
+cd D:/git/github/rl_institut/
```
Comment (Bachibouzouk) [inline snakemake/README.md]:
```suggestion
* git bash
move to the directory where you would like to clone the repository
```git 
```

File: snakemake/README.md [hunk 3]
```diff
@@ -10,6 +10,24 @@ Let's cook some Spätzle!
...
+git clone git@github.com:rl-institut/workshop.git
+cd workshop
+git pull
+```
+    
+* cmd
+```cms
+cd D:/git/github/rl_institut/workshop/snakemake
```
Comment (Bachibouzouk) [inline snakemake/README.md]:
```suggestion
* cmd
move to the `snakemake` folder inside the cloned `workshop` repository
```cms
```

**Review by Bachibouzouk — APPROVED** [review]
Thanks for the add on @Ludee, I made suggestions to make it more general and new user friendlier, you can choose to implement them or not :)

**Review by nesnoj — APPROVED** [review]
I cannot really comment on windows related stuff, but I trust you that it works :)

**Pull request merged**